CONTRIBUTING.md

```diff
@@ -334,6 +334,11 @@ new tag to the remote repository. This, in turn, triggers the _Release_ workflow
 on the CI, which builds and publishes the packages to NPM (with `pnpm publish`)
 and deploys the Storybook site.
 
+> `pnpm publish` triggers a `prepublish` script that removes the `type` field
+> from `package.json` before publishing each package. The reason for this
+> workaround is explained in
+> [#1219](https://github.com/silx-kit/h5web/issues/1219).
```

Comment:
Since we are at it, we may as well remind that this process changes the `package.json` via `publishConfig` since this is not obvious from the get-go.

Reply:
https://github.com/silx-kit/h5web/pull/1351/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R337-R350